src/app/navbar/navbar.component.ts

```diff
@@ -0,0 +1,115 @@
+import { Component, Injector, OnInit } from '@angular/core';
+import { slideMobileNav } from '../shared/animations/slide';
+import { MenuComponent } from '../shared/menu/menu.component';
+import { MenuService } from '../shared/menu/menu.service';
+import { MenuID, MenuItemType } from '../shared/menu/initial-menus-state';
+import { TextMenuItemModel } from '../shared/menu/menu-item/models/text.model';
+import { LinkMenuItemModel } from '../shared/menu/menu-item/models/link.model';
+import { HostWindowService } from '../shared/host-window.service';
+
+/**
+ * Component representing the public navbar
+ */
+@Component({
+  selector: 'ds-navbar',
+  styleUrls: ['navbar.component.scss'],
+  templateUrl: 'navbar.component.html',
+  animations: [slideMobileNav]
+})
+export class NavbarComponent extends MenuComponent implements OnInit {
+  /**
+   * The menu ID of the Navbar is PUBLIC
+   * @type {MenuID.PUBLIC}
+   */
+  menuID = MenuID.PUBLIC;
+
+  constructor(protected menuService: MenuService,
+              protected injector: Injector,
+              public windowService: HostWindowService
+  ) {
+    super(menuService, injector);
+  }
+
+  ngOnInit(): void {
+    this.createMenu();
+    super.ngOnInit();
+  }
+
+  /**
+   * Initialize all menu sections and items for this menu
+   */
+  createMenu() {
```

Comment:
Same note here as the Admin navbar. It'd be nice to understand the plan for this navbar configuration.

Reply:
Same answer 😄 
https://github.com/DSpace/dspace-angular/pull/338#discussion_r242848113